server/re/src/irods_re_serialization.cpp

```diff
@@ -1199,6 +1199,77 @@ namespace irods::re_serialization
         }
     } // serialize_vector_of_strings_ptr
 
+    static irods::error serialize_execMyRuleInp_ptr(boost::any _p, serialized_parameter_t& _out)
+    {
+        try {
+            log_re::trace(__func__);
+            const auto* casted_ptr = boost::any_cast<execMyRuleInp_t*>(_p);
+            if(!casted_ptr) {
+                _out["execMyRuleInp_ptr"] = "nullptr";
+                return SUCCESS();
+            }
+            _out["myRule"] = casted_ptr->myRule;
+            _out["addr_hostAddr"] = casted_ptr->addr.hostAddr;
+            _out["addr_zoneName"] = casted_ptr->addr.zoneName;
+            _out["addr_portNum"] = boost::lexical_cast<std::string>(casted_ptr->addr.portNum);
+            _out["addr_dummyInt"] = boost::lexical_cast<std::string>(casted_ptr->addr.dummyInt);
+            serialize_keyValPair(casted_ptr->condInput, _out);
+            _out["outParamDesc"] = casted_ptr->outParamDesc;
+
+            const auto* msparamary_ptr = casted_ptr->inpParamArray;
+            if(!msparamary_ptr) {
+                _out["inpParamArray"] = "nullptr";
+                return SUCCESS();
+            }
+            _out["inpParamArray_len"] = boost::lexical_cast<std::string>(msparamary_ptr->len);
+            _out["inpParamArray_oprType"] = boost::lexical_cast<std::string>(msparamary_ptr->oprType);
+            for (int i = 0; i < msparamary_ptr->len; i++) {
+                std::string prefix = "inpParamArray_" + std::to_string(i) + "_";
```

Comment:
Use `fmt::format()` or `+=` to concat strings. This avoids unnecessary temporaries.